lldb/source/Commands/CommandObjectCommands.cpp

```diff
@@ -1827,6 +1838,15 @@ class CommandObjectScriptingObjectParsed : public CommandObjectParsed {
 
   ScriptedCommandSynchronicity GetSynchronicity() { return m_synchro; }
 
+  std::optional<std::string> GetRepeatCommand(Args &args,
+                                              uint32_t index) override {
+    ScriptInterpreter *scripter = GetDebugger().GetScriptInterpreter();
+    if (!scripter)
+      return std::nullopt;
+
+    return scripter->GetRepeatCommandForScriptedCommand(m_cmd_obj_sp, args);
+  }
+
```

Comment:
nit: This looks pretty much the same as the one in `CommandObjectScriptingObjectRaw`. I understand that these have different base classes but I guess we could use multiple inheritance (in a follow-up) to refactor the 2 classes and reduce code duplication.

Reply:
I was holding off on refactoring these classes till we're switching to the Scripted Interface infrastructure.